wikihow2zim/scraper.py

```diff
@@ -29,9 +38,73 @@ def __init__(self, **kwargs):
         for option in options:
             setattr(self, option, kwargs[option])
 
+        if self.output_dir is None:
```

Comment:
This should never be `None` as you have a default coming from argparse.

Reply:
@rgaudin You're right :) I don't know how I didn't see that.